department-of-veterans-affairs/caseflow
# APPEALS-17253

Resolves [APPEALS-17253](https://vajira.max.gov/browse/APPEALS-17253)

### Description
Redirect to completed tab after 'documents ready for review' modal is submitted for Program Office -> VHA CAMO and VISN -> Program office. 

(the other AC were in place already)

### Acceptance Criteria
- [x] The following occurs when the "Documents ready for VHA CAMO review" task action is executed by a Program Office user
    - [x] The appeal appears on the ‘Completed’ tab of the VHA Program Office queue
    - [x] The appeal appears on the ‘Assigned’ tab of the VHA CAMO queue
    - [x] User is navigated to the ‘Completed’ tab in the Program Office queue
        - [x] A message appears confirming the case has been sent to VHA CAMO for review
- [x] The following occurs when the "Documents ready for Program Office review" task action is executed by a VISN user
    - [x] The appeal appears on the ‘Completed’ tab of the VHA VISN queue
    - [x] The appeal appears on the ‘Ready for Review’ tab of the VHA Program Office organization queue
    - [x] User is navigated to the VHA VISN organization queue ‘Completed’ tab
        - [x] A message appears confirming the case has been sent to the Program Office for review

### Test Data Creation

1. Open a rails console:  `rails c`
2. Generate some Program Office cases: `FactoryBot.create_list(:assess_documentation_task, 10, assigned_to: Organization.find_by(name: "Prosthetics"))`


### Testing Plan
1. Sign in as VHAPOUSER
2. Navigate to the Prosthetics queue
3. Assign a case to a VISN (you'll see it again in step 10)
4. Click one of the other cases and open the 'Documents ready for VHA CAMO review' task action
5. Enter anything and submit it - you should be redirected to the completed tab
6. Ensure the case is in the completed tab
7. Sign in as CAMOUSER
8. Navigate to the VHA CAMO queue and ensure the case is in their assigned tab
9. Sign in as VISNUSER and navigate to the queue for the VISN you chose in step 3
11. Click the case you assigned in step 3 and open the 'Documents ready for Program Office review' task action
12. Enter anything and submit it - you should be redirected to the completed tab
